app/scripts/constants.js

```diff
@@ -67,6 +67,8 @@ angular.extend(window.OPENSHIFT_CONSTANTS, {
     "Latest Release":          "https://github.com/openshift/origin/releases/latest"
   },
 
+  TECTONIC_URL: "https://redhat.com",
```

Comment:
Let's remove this from constants.js for now so no one tries to set it with an extension.